### PR TITLE
Add FaaS attributes.

### PR DESCRIPTION
**Pull request by anuraaga**

### CODE REVIEW

File: api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributes.java [hunk 1]
```diff
@@ -291,5 +291,47 @@
...
+  public static final StringAttributeSetter FAAS_INVOKED_NAME =
+      StringAttributeSetter.create("faas.invoked_name");
+  /** The cloud provider of the invoked function. */
+  public static final StringAttributeSetter FAAS_INVOKED_PROVIDER =
+      StringAttributeSetter.create("faas.invoked_provider");
+  /** The cloud region of the invoked function. */
+  public static final StringAttributeSetter FAAS_INVOKED_REGION =
+      StringAttributeSetter.create("faas.invoked_region");
+
+  // faas.trigger == datasource
```
Comment (Oberon00) [inline api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributes.java:313]: Should this comment be added to the Javadoc of the respective attributes somehow?

Comment (anuraaga) [reply api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributes.java:313]: Good idea, thanks

**Review by iNikem — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Comment by jkwatson** [comment]
Thanks! I bet you did this just so I would have to update #1631 , didn't you?  ;)

**Review by jkwatson — APPROVED** [review]
👍

**Pull request merged**